go.mod

```diff
@@ -26,24 +26,23 @@ require (
 	google.golang.org/genproto v0.0.0-20200731012542-8145dea6a485
 	google.golang.org/grpc v1.31.0
 	google.golang.org/protobuf v1.25.0
-	k8s.io/api v0.18.7-rc.0
-	k8s.io/apimachinery v0.18.7-rc.0
+	k8s.io/api v0.18.8
+	k8s.io/apimachinery v0.18.8
 	k8s.io/client-go v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
-	knative.dev/eventing v0.17.3
-	knative.dev/pkg v0.0.0-20200908235250-56fba14ba7df
-	knative.dev/serving v0.17.1-0.20200909134250-89faac4884b9
-	knative.dev/test-infra v0.0.0-20200909211651-72eb6ae3c773 // indirect
+	knative.dev/eventing v0.17.1-0.20200914161402-2237b8b18606
+	knative.dev/pkg v0.0.0-20200914163902-ea9a9de80c98
+	knative.dev/serving v0.17.1-0.20200914160002-c85936755fcd
 	sigs.k8s.io/yaml v1.2.0
 )
 
 replace (
 	istio.io/api => istio.io/api v0.0.0-20200227213531-891bf31f3c32
 	istio.io/client-go => istio.io/client-go v0.0.0-20200227214646-23b87b42e49b
```

Comment:
not related to this PR, but is this istio dependency needed here?

Reply:
This is likely a serving dependency, but I'm not sure why it's a replace. Might be copied from serving?

Reply:
yeah, no idea where this comes from. Resolving this comment. let me know if furhter action is needed.